### ISSUE TITLE
HTTP method HEAD returns an error

### ISSUE THREAD

**Issue by pavelsaman** [firecampapp/Firecamp#60]
1. problem

When a resource is requested using `HEAD` HTTP method, the following error is returned:

![image](https://user-images.githubusercontent.com/24702416/73943580-462e9e80-48f1-11ea-8119-1e2d5d382766.png)

Also, all other tabs in response section are empty (Headers and Timeline in this case; in Timeline, where I'd expect to find info about the sent request), so it indicates there's something wrong with interpreting the response:

![image](https://user-images.githubusercontent.com/24702416/73944737-5e9fb880-48f3-11ea-8a87-ab6589954c2a.png)

2. steps to reproduce

tested against my own little Python server on localhost:

```python
from flask import Flask
from flask import Response

app = Flask(__name__)

@app.route('/')
def example():
    resp = Response('{"a": 5}')
    resp.headers['Content-Type'] = 'application/json'
    resp.headers['custom-header'] = 'abc'
    return resp

if __name__ == '__main__':
    app.run()
```

=> send a req with `HEAD` method:

![image](https://user-images.githubusercontent.com/24702416/73944214-7b87bc00-48f2-11ea-9f47-0e4ebe3aa6ad.png)

3. desired state

there're 5 headers I should be given when `HEAD` method is used:

![image](https://user-images.githubusercontent.com/24702416/73944148-61e67480-48f2-11ea-8233-6c686dfa30f3.png)

![image](https://user-images.githubusercontent.com/24702416/73944434-db7e6280-48f2-11ea-89b8-bfea05461878.png)

4. version

Firecamp v0.9.3 on Windows x64

**Comment by Nishchit14** [comment]
Thanks for driving attention, I'm checking this issue. will update you soon.

**Comment by pranavwani** [comment]
@pavelsaman , because of the default behavior of `CURL`, If the response contains `content-length` in the header and response body does not exist at that time this error throws.

![Screenshot from 2020-02-18 09-55-06](https://user-images.githubusercontent.com/26399261/74703959-d398b980-5234-11ea-91e0-3934dc84ccb8.png)

For more reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/HEAD

- To, execute this request you can enable `Ignore response content length` option from the config(Added in v1.0.1).

![Screenshot from 2020-02-18 10-02-37](https://user-images.githubusercontent.com/26399261/74704296-d34cee00-5235-11ea-9d17-2914ea1ef370.png)

**Comment by Nishchit14** [comment]
Thank you @pranavwani for sharing the possible solution 👏, I hope you'll be on the same understanding @pavelsaman. please suggest your thoughts if we can improve it in another way.

**Comment by Nishchit14** [comment]
released under [v1.0.1](https://github.com/firecampapp/Firecamp/tree/1.0.1)

@pavelsaman feel free to open it if you still think that It would have better solution, Thanks again for your support.
